discovery-infra/tests/test_cancel_and_reset.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 from tests.base_test import BaseTest
+from tests.conftest import env_variables
```

Comment:
can't we use it as Base test env?

Reply:
not that im aware of, any suggestions?

Reply:
i think better to have func in base test that will return envs